/usr/share/abuild/default.conf adds these 2 flags to CFLAGS,
but they are not recognized by Slimcc, so add them to the
ignore list.

--- a/main.c
+++ b/main.c
@@ -513,6 +513,8 @@ static int parse_args(int argc, char **argv) {
         !strcmp(argv[i], "-fno-stack-protector") ||
         !strcmp(argv[i], "-fno-strict-aliasing") ||
         !strcmp(argv[i], "-fno-strict-overflow") ||
+        !strcmp(argv[i], "-fstack-clash-protection") ||
+        !strcmp(argv[i], "-fno-plt") ||
         !strcmp(argv[i], "-fwrapv") ||
         !strcmp(argv[i], "-m64") ||
         !strcmp(argv[i], "-mfpmath=sse") ||
